lib/versioning/pvp/readme.md

```diff
@@ -0,0 +1,21 @@
+# PVP
+
+## What type of versioning is used?
+
+Quotes from the [Haskell PVP Specification](https://pvp.haskell.org/):
+
+> A package version number **SHOULD** have the form A.B.C, and **MAY** optionally have any number of additional components, for example 2.1.0.4 (in this case, A=2, B=1, C=0).
+> A.B is known as the _major_ version number, and C the _minor_ version number.
```

Comment:
```suggestion

> A.B is known as the _major_ version number, and C the _minor_ version number.
```

These are separate quotes in the upstream docs, so we should put a newline between them.